src/test/java/com/jcabi/w3c/DefaultCssValidatorTest.java

```diff
@@ -82,4 +85,23 @@ public void ignoresEntireDocument() throws Exception {
         MatcherAssert.assertThat(response.toString(), response.valid());
     }
 
+    /**
+     * DefaultHtmlValidator throw IOException when W3C server is unavailable.
+     * @throws Exception If something goes wrong inside
+     * @todo #10:30min DefaultCssValidator have to be updated to throw only
+     *  IOException when W3C validation server is unavailable. Any other
+     *  exception type can be confusing for users. Remove @Ignore annotation
+     *  after finishing implementation.
+     */
+    @Ignore
+    @Test(expected = IOException.class)
+    public void throwsIOExceptionWhenValidationServerIsUnavailable()
+        throws Exception {
+        final MkContainer container = new MkGrizzlyContainer().next(
+            new MkAnswer.Simple(HttpURLConnection.HTTP_UNAVAILABLE)
```

Comment:
@dmzaytsev As far as I see the `MkContainer` can return only one status at once. According to [wikipedia](https://en.wikipedia.org/wiki/List_of_HTTP_status_codes#5xx_Server_Error) there are ten 50x errors. There are two options to test them all:
1. write separate test for each error code
2. update existing test to return one random 50x status
what do you suggest here?